front/src/applications/editor/tools/pointEdition/tool-factory.tsx

```diff
@@ -80,6 +83,35 @@ function getPointEditionTool<T extends EditorPoint>({
           },
         },
       ],
+      [
+        {
+          id: 'delete-entity',
+          icon: BsTrash,
+          labelTranslationKey: `Editor.tools.${id}-edition.actions.delete-entity`,
+          // Show button only if we are editing
+          isDisabled({ state }) {
+            return state.initialEntity.geometry.type === 'GeometryCollection';
+          },
+          onClick({ infraID, openModal, closeModal, forceRender, state, setState, dispatch, t }) {
+            openModal(
+              <ConfirmModal
+                title={t(`Editor.tools.${id}-edition.actions.delete-entity`)}
+                onConfirm={async () => {
+                  await dispatch<ReturnType<typeof save>>(
```

Comment:
I think the await is not useful here ?
```suggestion
                  await dispatch<ReturnType<typeof save>>(
```

Reply:
Tried to remove it but the save function seems to be asynchronous. If it's not there, the deleted object is still displayed on the map untill we switch tool

Reply:
Try to remove async from `onConfirm={async () => {`, onConfirm take either a async function or normal fonction, i think, if you passe an async function but without await it will not be executed

Reply:
Okay fine for me :+1: